quickstart.rb

```diff
@@ -0,0 +1,53 @@
+require 'google/apis/gmail_v1'
```

Comment:
This is a google supplied program that you have to have. Literally they tell you to copy and paste it.